components/quick_switch_modal/quick_switch_modal.jsx

```diff
@@ -165,7 +165,7 @@ export default class QuickSwitchModal extends React.PureComponent {
 
     handleSuggestionsReceived = (suggestions) => {
         const noLoadingProp = suggestions.items.some((item) => !item.loading);
-        this.setState({hasSuggestions: !suggestions.matchedPretext || (suggestions.items.length > 0 && noLoadingProp), pretext: suggestions.matchedPretext});
+        this.setState({hasSuggestions: (suggestions.items.length > 0 && noLoadingProp) || !suggestions.matchedPretext, pretext: suggestions.matchedPretext});
```

Comment:
I'm not sure if this was intended to change anything, but it just reorders the condition without changing the behaviour, so that would be why Jelena didn't see anything change in the behaviour